Frontend/ui-customer/src/Services/EndPoint.purs

```diff
@@ -308,4 +308,22 @@ getManuallySharedRideDetails :: String -> String
 getManuallySharedRideDetails rideId = (getBaseUrl "60") <> "/followRide/" <> rideId <> "/customerDetails"
 
 multiChat :: String -> String 
-multiChat _ = (getBaseUrl "61") <> "/triggerFCM/message"
\ No newline at end of file
+multiChat _ = (getBaseUrl "61") <> "/triggerFCM/message"
+
+getFavouriteDriverList :: String
+getFavouriteDriverList = ((getBaseUrl "58") <> "/driver/favorites") 
+
+getFavouriteDriverTrips :: String -> String -> String -> Maybe String -> Maybe String -> String
```

Comment:
@ROHIT-43 Can you explain what is the logic here

Reply:
first 3 values are mandatory so they cannot be nothing last 2 values is of Maybe type so first this logic is checking that is status is null or not if it is not null then it will check that if clientId is null or not and depends on this it will call the endpoint